woocommerce/wc-admin
# Add note on first order creation

Fixes #4222 

Adds a note to the inbox on first order creation.

Note that `paginate` and `return` params were removed from the `WC_Order_Query` since these produced unexpected results with drafts and the total (though this could be specific to my install).

/cc @pmcpinto This only creates a note for the user when an order is created via the checkout page, but we could alternatively add this note even if they manually add an order in the dashboard.

### Screenshots
<img width="512" alt="Screen Shot 2020-06-09 at 3 16 41 PM" src="https://user-images.githubusercontent.com/10561050/84146475-708d9580-aa64-11ea-9638-2f06110760f4.png">

### Detailed test instructions:

1. Create a fresh site with no orders (note that I ran into issues trying to manually do this with an older site that had refunds and other order types lying around).
1. Create an order via the front-end.
1. Note the note.
1. On an older site with orders, repeat this process and make sure a note is not added.
